### PR TITLE
Add checkbox to include loc ids in export

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?186316
@sravfeyn

### CODE REVIEW

File: corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js [hunk 1]
```diff
@@ -10,15 +10,18 @@ $(function () {
...
         this.include_consumption = ko.observable(false);
+        this.include_ids = ko.observable(false);
         self = this;
         this.url = ko.computed(function() {
-            // ternary prevents adding include_consumption=false to other
-            // bulk pages
-            return self.base_url + (self.include_consumption() ? "?include_consumption=true" : "");
+            return (
+                self.base_url
+                + (self.include_consumption() ? "?include_consumption=true" : "")
```
Comment (dimagimon) [inline corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js]: Bad line breaking before '+'.

File: corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js [hunk 2]
```diff
@@ -10,15 +10,18 @@ $(function () {
...
+                + (self.include_ids() ? "?include_ids=true" : "")
```
Comment (dimagimon) [inline corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js]: Bad line breaking before '+'.

File: corehq/apps/locations/util.py [hunk 3]
```diff
@@ -184,6 +185,7 @@ def _loc_type_dict(self, loc_type):
             uncategorized_keys.update(uncategorized_data.keys())
 
             loc_dict = {
+                'location_id': loc.location_id,
```
Comment (sravfeyn) [inline corehq/apps/locations/util.py]: Should `loc_dict` contain `location_id` irrespective of whether `include_ids` is `False`

Comment (esoergel) [inline corehq/apps/locations/util.py]: I think it's fine - it only gets put in the export if it corresponds to a header in `tab_headers`.  It's small and cheap enough to compute that I don't think it'd be an issue.  (I tested it out both ways when I wrote this to be sure).

Comment (esoergel) [inline corehq/apps/locations/util.py]: Oh, and `consumption` is also in there regardless of whether or not it's used (the value is just `{}` if not relevant, 'cause that one IS expensive to compute).

Comment (sravfeyn) [inline corehq/apps/locations/util.py]: Cool, makes sense

**Pull request merged**